.ci.yaml

```diff
@@ -3212,17 +3160,6 @@ targets:
       task_name: tiles_scroll_perf_ios__timeline_summary
     scheduler: luci
 
-  - name: Mac_ios32 flutter_gallery__transition_perf_e2e_ios32
```

Comment:
I don't see the 32bit versions running on https://ci.chromium.org/p/flutter/g/devicelab_benchmark/console#.

Reply:
Did we decide to stop running those? @jmagman

Reply:
This is not intended. Have restored the test. Thanks!